discord/bot.py

```diff
@@ -445,8 +445,7 @@ def command(self, **kwargs):
         """There is an alias for :meth:`application_command`.
 
         .. note::
-
```

Comment:
Why is this newline deleted?

Reply:
Newline isn't needed there

Reply:
There is a newline under every `.. note` in the source.

Reply:
okithen